kuma/settings/pytest.py

```diff
@@ -102,3 +102,6 @@
 # our tests to not deviate when the actual number changes since that number
 # change shouldn't affect the tests.
 CONTRIBUTION_AMOUNT_USD = 4.99
+
+SENDINBLUE_API_KEY = "testing"
```

Comment:
Yikes! Sorry. I can't believe I missed this in the review.

Reply:
No don't be, I can't believe I missed doing it!